packages/astro/src/assets/utils/svg.ts

```diff
@@ -5,7 +5,9 @@ import type { ImageMetadata } from '../types.js';
 
 function parseSvg(contents: string) {
 	const root = parse(contents);
-	const [{ attributes, children }] = root.children;
+	const { attributes, children } = root.children.find(
+		({ name, type }: { name: string; type: number }) => type === 1 /* Element */ && name === 'svg',
+	);
```

Comment:
(optional) perhaps we should also handle the case where we can't find any svg element

Reply:
I agree as well